omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/MockModuleFactoryWrapper.java

```diff
@@ -76,6 +76,13 @@ public Module loadModule(File moduleFile) {
 	}
 	
 	@Override
+	public Module updateModule(Module module, String downloadUrl) {
+		module.setDownloadURL(downloadUrl);
+		module.setVersion("2.2");
```

Comment:
Why is this hard coded here? I'm guessing this is the version the client wishes to upgrade to, right?

Reply:
Oops! Sorry just noticed this is a mock